setup.py

```diff
@@ -1,24 +1,23 @@
-import os
-from setuptools import setup, find_packages
 from pathlib import Path
 
-requirements = {}
-for extra in ["docs", "main"]:
-    requirements[extra] = [
-        r for r in Path(f"requirements/{extra}.txt").read_text().splitlines()
-    ]
+from setuptools import setup, find_packages
+
+
+def read_requirements(path):
+    return list(Path(path).read_text().splitlines())
+
 
-setup(name='u8timeseries',
-      version=open("u8timeseries/VERSION", "r").read(),
+setup(
+      name='u8timeseries',
+      version=open('u8timeseries/VERSION', 'r').read(),
       description='A collection of easy-to-use timeseries forecasting models',
       url='http://github.com/unit8co/u8timeseries',
       author='Unit8 SA',
       author_email='info@unit8.co',
       license='Apache License 2.0',
       packages=find_packages(),
-      install_requires=requirements["main"],
-      extras_require={"docs": requirements["docs"]},
+      install_requires=read_requirements('requirements/main.txt'),
```

Comment:
@endrjuskr will this work without extras_require?

Reply:
just pointing out that the github actions do work with this change